examples/interpolation/stan/build_b_spline_example.stan

```diff
@@ -0,0 +1,71 @@
+// Copywrite Milad Kharratzadeh with modification suggested by Martin Modrak
+// https://github.com/milkha/Splines_in_Stan
+// Accessed Feb. 9, 2021 
+// modification suggested in https://github.com/milkha/Splines_in_Stan/issues/3#issuecomment-319671070
+
+function {
+  #include build_b_spline.stan
```

Comment:
all the functions should end with `.stanfunctions`. You wouldn't have known this, so no worries! It's a new feature in the next Stan release and this repo will go "live" shortly after.